translations/en.json

```diff
@@ -2020,6 +2020,15 @@
     "bold": "Bold",
     "italic": "Italic",
     "strikethrough": "Strikethrough",
+    "empty-notifications-all-tab": "No unread notifications.",
+    "empty-notifications-admin-tab": "No unread admin notifications.",
+    "empty-notifications-mentions-tab": "No unread mention notifications.",
+    "empty-notifications-replies-tab": "No unread reply notifications to your messages.",
+    "empty-notifications-contact-requests-tab": "No contact requests notifications",
+    "empty-notifications-identity-verification-tab": "No identity verification requests",
+    "empty-notifications-transactions-tab": "No transaction notifications",
+    "empty-notifications-membership-tab": "No membership notifications",
+    "empty-notifications-system-tab": "No system notifications",
```

Comment:
Some values have a dot (full stop/period) at the end of the sentence. Please check the copy.

Reply:
yeah odd, perhaps it's where I copied them from. I'll tag you in the chat with these messages